### ISSUE TITLE
Filters heading is only available to screen reader users

### ISSUE THREAD

**Issue by sachajudd** [silverstripe/cwp-starter-theme#154]
**Medium priority:**
There is a h2 "Filters" heading that is only available to screen reader users. This heading should be provided for all users or removed.
![image](https://user-images.githubusercontent.com/24258161/60930692-b862e480-a30a-11e9-9dcc-abdd892982c2.png)
**Impact:** 
Users reliant on the page markup and heading structure of a page to relate content to each other hierarchically will not be able to do so.
**Solution:**
Remove the sr-only class from the "Filters" heading so that it is available to all users. Example:  ```<h2 class="sr-only">Filters</h2>```

**Comment by sachajudd** [comment]
cc @silverstripeux

**Comment by clarkepaul** [comment]
All good, let's reduce the visual size of the headers then but leave a good semantic structure. We would probably remove the word "Filter" from the minor headings.

**Comment by sachajudd** [comment]
Something like this?
![image](https://user-images.githubusercontent.com/24258161/61097329-221bf380-a4af-11e9-8cb2-622ab9c1c575.png)

**Comment by ScopeyNZ** [comment]
Closed by #156
